package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "uncrustify",
-  "version": "0.6.1",
+  "version": "0.64",
   "description": "A highly configurable, easily modifiable source code beautifier for C, C++, C#, ObjectiveC, D, Java, Pawn and VALA",
   "main": "uncrustify",
   "scripts": {
-      "preinstall": "./configure && make"
+      "preinstall": "mkdir build; cd build; cmake .."
   },
   "bin" : "src/uncrustify" ,
   "repository": {
     "type": "git",
-    "url": "https://github.com/bengardner/uncrustify"
+    "url": "https://github.com/uncrustify/uncrustify"
   },
   "keywords": [
     "uncrustify",
     "source code beautifier"
   ],
   "author": "Ben Gardner",
   "license": "GPLv3"
```

Comment:
I am uncertain about this. This repo states GPLv2 is used. inside the file [uncrustify.cpp](https://github.com/uncrustify/uncrustify/blob/d0d3a32b5ce1e871905305f05012554995c0b588/src/uncrustify.cpp#L6) GPLv2**_+_** is mentioned. 
Does that mean Uncrustify is licensed under `GPLv[2, ∞)` ?